contrib/download-frozen-image-v2.sh

```diff
@@ -301,6 +301,7 @@ while [ $# -gt 0 ]; do
 			-H 'Accept: application/vnd.docker.distribution.manifest.v2+json' \
 			-H 'Accept: application/vnd.docker.distribution.manifest.list.v2+json' \
 			-H 'Accept: application/vnd.docker.distribution.manifest.v1+json' \
+			-H 'Accept: application/vnd.oci.image.manifest.v1+json' \
```

Comment:
Same order swap and OCI index addition: :eyes:

```suggestion
			-H 'Accept: application/vnd.oci.image.manifest.v1+json' \
			-H 'Accept: application/vnd.oci.image.index.v1+json' \
			-H 'Accept: application/vnd.docker.distribution.manifest.v2+json' \
			-H 'Accept: application/vnd.docker.distribution.manifest.list.v2+json' \
			-H 'Accept: application/vnd.docker.distribution.manifest.v1+json' \
			-H 'Accept: application/vnd.oci.image.manifest.v1+json' \
```